content/articles/introduction-to-c/index.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: engineering-education
+status: draft
+published: false
+url: /introduction-to-c/
+title: Introduction to C programming
+description: It covers the history of C, its purpose, where it is used, and how to install and files used in the C and First C program.
+author: chandramouli-dasari
+date: 2021-06-03T00:00:00-08:10
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/introduction-to-c/hero.png
+    alt: Clang image example
+---
+
+![Clang image example](/engineering-education/introduction-to-c/hero.png)
+
+
+
+### Introduction
+
+C is the one of most popular languages. C is a practical and basic programming language. This "C" is used as a reference for other languages in computer science courses. And as per competitive programming, it is also the most preferable language. When they starting programming or shifting from another programming language then "C" looks like bit difficult 
+but this series will explain everything from ground level to high level
+
+### Table of contents
+
+- History of C Program
+- Structure of C program
+- Installing C in our System
+- Writing the First C program
+- Files used in C program
+- Uses Of C programming
+
+### History of C Program
+
+C is a practical programming language. In the 1970's "C" was developed by Dennis Ritchie and used by the UNIX operating system at Bell Laboratories.
+
+- Why this language named C?
+  - Before "C" there is a language called "B" and Some of the Options are derived from "B"
+- What is the purpose of the C language?
+  - Early "C" is only used to design and implement the system software. But later some features were add to "C" then it is used to develop portable application software too.
+
+
+### Structure of C program
+
+`C` program is structured of preprocessor commands, a global declaration section, and one or more functions
+
+```
+preprocessor directives
+
+Global declarations
+main()
+{
+ Local declarations
+ Statements
+ }
+ Function 1()
+{
+ Local declarations
+ Statements
+ }
+ Function N()
+{
+ Local declarations
+ Statements
+ }
+```
```

Comment:
Could you explain the code above

Reply:
And that was not the code it is just a Structure